.circleci/config.yml

```diff
@@ -703,11 +736,19 @@ jobs:
       release:
         type: boolean
         default: false
+      push_nightly:
+        type: boolean
+        default: false
     executor: oss-linux
-    resource_class: xlarge
+    resource_class: large
```

Comment:
this does slow down the build considerably but it might be OK since the build is fast-ish now.